src/styles/utils/variables.scss

```
@@ -0,0 +1,3 @@
+$cell-size: 100px;
+$calendar-width: $cell-size * 7 + 7px;
```

do not use magic numbers like 7px
```suggestion
$calendar-width: ($cell-size + $gap-between-size) * 7;
```